cmake/pkgconfig.in

```diff
@@ -6,6 +6,6 @@ Name: libOSRM
 Description: Project OSRM library
 Version: v@OSRM_VERSION_MAJOR@.@OSRM_VERSION_MINOR@.@OSRM_VERSION_PATCH@
 Requires:
-Libs: -L${libdir} -losrm
+Libs: -L${libdir} -losrm @CMAKE_EXE_LINKER_FLAGS@
```

Comment:
I might be a bad idea to expose internal linker flags here, because these environment variable can include settings that are irrelevant to external users.

Controlling this explicitly over a new variable `OSRM_LIB_LINKER_FLAGS` (or some other name) would make sure we only get flags (like the rpath) that we want external users to know about.